spec/models/item_spec.rb

```diff
@@ -24,4 +28,28 @@
     end
   end
 
+  describe '.valid_items' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.